File: samples/java-spring-file-writer/src/main/java/art/galushko/java/spring/file/writer/UsersOrdersApplication.java
package art.galushko.java.spring.file.writer;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.ImportAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.validation.ValidationAutoConfiguration;

@SpringBootApplication
@ImportAutoConfiguration(ValidationAutoConfiguration.class)
public class UsersOrdersApplication {
    public static void main(String[] args) {
        SpringApplication.run(UsersOrdersApplication.class, args);
    }
}

File: samples/kotlin-spring-rest-assured/src/main/kotlin/art/galushko/kotlin/spring/rest/assured/UsersOrdersApplication.kt
package art.galushko.kotlin.spring.rest.assured

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.ImportAutoConfiguration
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.validation.ValidationAutoConfiguration

@SpringBootApplication
@ImportAutoConfiguration(ValidationAutoConfiguration::class)
open class UsersOrdersApplication

fun main(args: Array<String>) {
    SpringApplication.run(UsersOrdersApplication::class.java, *args)
}
